src/hooks/useUser.ts

```diff
@@ -25,15 +26,14 @@ export interface User {
 
 type NotificationAgentTypes = Record<NotificationAgentKey, number>;
 
-export interface UserSettings {
-  discordId?: string;
-  region?: string;
-  originalLanguage?: string;
-  locale?: string;
+type UserSettings = Pick<
```

Comment:
I noticed here that we're duplicating code and having multiple sources of truth. This Pick type allows us to honor the types already set for these settings in the UserSettingsGeneralResponse interface. If it's desired to have it typed out explicitly in this file though, we can simply remove what I added and instead add `hideTags?: boolean;` underneath line 32 of the original code.